code/modules/mob/living/carbon/human/human.dm

```diff
@@ -431,6 +431,38 @@
 				to_chat(usr, "<b>Notes:</b> [R.fields["notes"]]")
 				return
 
+			if(href_list["add_citation"])
+				if(!H.canUseHUD())
+					return
+				if(!HAS_TRAIT(H, TRAIT_SECURITY_HUD))
+					return
+				var/t1 = stripped_input("Please input citation crime:", "Security HUD", "", null)
+				var/fine = FLOOR(input("Please input citation fine:", "Security HUD", 50) as num|null, 1)
+				if(isnull(fine))
+					return
+				if(fine < 0)
+					to_chat(usr, "<span class='warning'>You're pretty sure that's not how money works.</span>")
+					return
+				fine = min(fine, 1000) //maxFine in the regular sec thing. pretty sure this should be a referece to that somehow.

```

Comment:
maxFine in the security console doesn't _feel_ like it should be a local variable, should it be defined as one of those static global variables like MAX_FINE?

Reply:
How would people feel about this being in server config? Unnecessary or justified?

Reply:
if you want to go to the extra effort, I won't stop you

Reply:
epic